### ISSUE TITLE
Error message appears when training CRAFT with SynthText dataset from scratch?

### ISSUE THREAD

**Issue by daigang896** [JaidedAI/EasyOCR#946]
Hello，
I don't know how the size inconsistency occurs when training CRAFT with SynthText dataset from scratch?
The training script is: CUDA_VISIBLE_DEVICES=0 python3 trainSynth.py --yaml=syn_train

The information is as follows:
2023-02-08:01:38:12, training_step: 49950|50000, learning rate: 0.00000080, training_loss: 0.03255, avg_batch_time: 29187.57935
2023-02-08:01:38:14, training_step: 49955|50000, learning rate: 0.00000080, training_loss: 0.03671, avg_batch_time: 29189.06155
Error : operands could not be broadcast together with shapes (200,200) (23,0) 
On generating affinity map, strange box came out. (width: 0, height: 23)

I don't know if anyone can help solve this problem. Thank you.

**Comment by rksamal-bh** [comment]
i have the same problem if anyone has found out the issue please tell

**Comment by rksamal-bh** [comment]
> Hello， I don't know how the size inconsistency occurs when training CRAFT with SynthText dataset from scratch? The training script is: CUDA_VISIBLE_DEVICES=0 python3 trainSynth.py --yaml=syn_train
> 
> The information is as follows: 2023-02-08:01:38:12, training_step: 49950|50000, learning rate: 0.00000080, training_loss: 0.03255, avg_batch_time: 29187.57935 2023-02-08:01:38:14, training_step: 49955|50000, learning rate: 0.00000080, training_loss: 0.03671, avg_batch_time: 29189.06155 Error : operands could not be broadcast together with shapes (200,200) (23,0) On generating affinity map, strange box came out. (width: 0, height: 23)
> 
> I don't know if anyone can help solve this problem. Thank you.

do you know where the weights file  are stored

**Comment by xu-peng-7** [comment]
Firstly, the reasons for this type of error are all from the generation of affinity maps and box generation errors. The specific error is that after processing the map obtained by the model, two boxes are obtained, which are the predicted two chars. After passing through these two boxes, a box is generated, which is the middle part of the two chars used to generate the affinity map. At this point, if the generated box is a single column, but the calculation is based on one row, it will result in its width being zero.

**Comment by Talhaz** [comment]
> Firstly, the reasons for this type of error are all from the generation of affinity maps and box generation errors. The specific error is that after processing the map obtained by the model, two boxes are obtained, which are the predicted two chars. After passing through these two boxes, a box is generated, which is the middle part of the two chars used to generate the affinity map. At this point, if the generated box is a single column, but the calculation is based on one row, it will result in its width being zero.

How it gonna solve ?
